docs/articles/keyboard-shortcuts.md

```diff
@@ -10,6 +10,7 @@ Files comes with built in keyboard shortcuts to make naviagting the app easier.
 | `Ctrl` + `V` | Paste |
 | `Ctrl` + `T` | Create new tab |
 | `Ctrl` + `W` | Close tab |
+| `Ctrl` + `F4` | Close tab |
```

Comment:
Is there a way to disable it?

Reply:
It seems like a valid shortcut even in apps like Edge so I don't think we should disable it in Files (it might be useful for othe apps), what would be nice however, was if we could change the tooltip in the Tab control to display `Ctrl+W`.

Reply:
Hmmm that's a valid point. Question is if either of those are better, i.e. more worthy of being the tooltip than the other. You can probably do some code-behind magic to update the tooltip. However, this would more or less mean that you would use localization.